bumble/gatt.py

```diff
@@ -187,7 +187,7 @@
 # -----------------------------------------------------------------------------
 
 
-def show_services(services):
+def show_services(services: List[Service]) -> None:
```

Comment:
since we only iterate over the services here, we could be more flexible an accept `Iterable[Service]` or `Sequence[Service]`